docs/index.md

```diff
@@ -1,31 +1,23 @@
 Zanata Server
 =============
+Please visit [our about page](http://zanata.org/about/)
 
-TBD
-
-Features
---------
```

Comment:
Maintainer's and users' guide should also be on readthedocs IMO. Anything that we may want to keep versioned should really go there. That way we can show different documentation for the most recent versions of Zanata.